sockeye/encoder_pt.py

```diff
@@ -90,14 +90,17 @@ class PyTorchEmbedding(PyTorchEncoder):
     def __init__(self, config: EmbeddingConfig, embedding: Optional[pt.nn.Embedding] = None) -> None:
         super().__init__()
         self.config = config
-
         if embedding is not None:
             self.embedding = embedding
         else:
             self.embedding = pt.nn.Embedding(self.config.vocab_size, self.config.num_embed,
                                              sparse=self.config.allow_sparse_grad)
 
+        self.num_factors = self.config.num_factors
+        if self.num_factors > 1:
+            assert len(self.config.factor_configs) == self.num_factors - 1
```

Comment:
Do we expect this to be violated?
If we want to keep it should it be converted to `check_condition` + a message?

Reply:
No, I think its safe to assume this is always true. I will remove the assertion.